requirements/base.txt

```diff
@@ -4,8 +4,8 @@
 #
 #    make upgrade
 #
-django-waffle==0.14.0
-django==1.11.15
-newrelic==4.2.0.100
+django-waffle==0.16.0
+django==1.11.20
```

Comment:
According to an email from Feanil, 1.11.21 was a security release.  I'm not sure why that wasn't picked up.  If you run again do you get it?

Reply:
The version release is due on June 3rd, 2019.